.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:9.11.1
+    environment:
+      REPO: "dddecaf/decaf-metabolica"
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: npm install
+      - run: ./.circleci/environment.sh npm run build
+      - run: ./.circleci/install_gcloud.sh
+      - run: ./.circleci/install_kubectl.sh
+      - run: docker build -f Dockerfile -t $REPO:$CIRCLE_BRANCH .
+      - run: docker tag $REPO:$CIRCLE_BRANCH $REPO:${CIRCLE_SHA1::12}
```

Comment:
FYI you can add all tags while building, f.ex. `docker build -t $REPO:$CIRCLE_BRANCH -t $REPO:${CIRCLE_SHA1::12} .`